Monika After Story/game/script-holidays.rpy

```diff
@@ -7150,7 +7178,7 @@ init 5 python:
             action=EV_ACT_UNLOCK,
             rules={"no_unlock": None},
             start_date=mas_monika_birthday,
-            end_date=mas_monika_birthday + datetime.timedelta(days=1),
+            end_date=datetime.datetime.combine(mas_monika_birthday+datetime.timedelta(days=1), datetime.time(hour=1)),
```

Comment:
just to confirm, this is because we want Monika's bday to end an hour beyond midnight, correct?

Reply:
correct